Yank/tests/test_yaml.py

```diff
@@ -1550,9 +1550,9 @@ def test_default_platform_precision():
             assert platform.getPropertyDefaultValue('CudaPrecision') == 'mixed'
         elif platform_name == 'OpenCL':
             if opencl_support_double:
-                assert platform.getPropertyDefaultValue('OpenCLPrecision') == 'mixed'
```

Comment:
What about using a `try...except` clause so that we can support old and new OpenMM versions for testing?